cast/src/starknet_commands/declare.rs

```diff
@@ -15,41 +15,114 @@ use starknet::{
     signers::LocalWallet,
 };
 use std::sync::Arc;
+use std::process::{Command, Stdio};
+use serde::Deserialize;
+use std::path::PathBuf;
+
+#[allow(dead_code)]
+#[derive(Deserialize)]
+struct ScarbStarknetArtifacts {
+    version: u32,
+    contracts: Vec<ScarbStarknetContract>,
+}
+
+#[allow(dead_code)]
+#[derive(Deserialize)]
+struct ScarbStarknetContract {
+    id: String,
+    package_name: String,
+    contract_name: String,
+    artifacts: ScarbStarknetContractArtifact,
+}
+
+#[allow(dead_code)]
+#[derive(Deserialize)]
```

Comment:
I think those structs should be moved to some new file. wdyt @THenry14 ?

Reply:
we can leave them here for now, as they are directly connected to declare. there is a possibility we'll move them in #2113